File: eRecipes/UI/erecipes_mobile/lib/models/radni_prostor.dart
import 'package:json_annotation/json_annotation.dart';

part 'radni_prostor.g.dart';

@JsonSerializable()
class RadniProstor {
  int? radniProstorId;
  String? oznaka;
  int? kapacitet;
  bool? aktivan;

  RadniProstor();

  factory RadniProstor.FromJson(Map<String, dynamic> json) =>
      _$RadniProstorFromJson(json);

 Map<String, dynamic> toJson() => _$RadniProstorToJson(this);
}

File: eRecipes/eRecipes.Model/RadniProstor.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eRecipes.Model
{
    public class RadniProstor
    {
        public int RadniProstorId { get; set; }
        public string Oznaka { get; set; }
        public int Kapacitet { get; set; }
        public bool Aktivna { get; set; }
    }
}
